CONTRIBUTING.md

```diff
@@ -1,3 +1,17 @@
 # Contributing
 
 See our specific contributing docs [here](https://smartcontractkit.github.io/chainlink-testing-framework/contributing/), along with the general [Chainlink contributing guidelines](https://docs.chain.link/docs/contributing-to-chainlink/).
+
+# Creating new PRs
+
+When creating a new PR remember to:
+* include the id of JIRA ticket in the format `[PROJECT_ID-ticket-id]` (e.g.`[TT-1234]` for a Test Tooling ticket with ID 1234) in the PR title
+* include a short description of the PR
+
+# Versioning
+When releasing a new version remember to follow correct semver:
+* `major` version when you make incompatible API changes,
+* `minor` version when you add functionality in a backwards compatible manner, and
+* `patch` version when you make backwards compatible bug fixes.
+
+You can read more abou semver [here](https://semver.org/).
```

Comment:
@Tofel could you add few lines about releasing new version on Github using https://github.com/smartcontractkit/chainlink-testing-framework/releases > Draft a new release ?